packages/xdl/src/User.ts

```diff
@@ -97,6 +97,10 @@ export class UserManagerInstance {
     this._getSessionLock = new Semaphore();
   }
 
+  constructor() {
+    this.getCachedUserDataAsync = this.getCachedUserDataAsync.bind(this);
```

Comment:
i prefer defining the function with fat arrow binding instead, less likely for this to break in the future and more self explanatory